common/persistence/tests/cassandra_test.go

```diff
@@ -25,6 +25,7 @@
 package tests
 
 import (
+	"go.uber.org/zap/zaptest"
```

Comment:
I don't see any existing dependencies on this package (I just grepped my source tree). I'm leary of adding a new third party dependency without a little justification.  Does zaptest add something special to justify depending on a new package?

Reply:
I actually added this dependency based on feedback in: https://github.com/temporalio/temporal/pull/2417#discussion_r793136654

Afaict this is a submodule of zap, which we already depend on.

Reply:
aha, hence my grep failing.  Lgtm.